### PR TITLE
Added new project 

### PR DESCRIPTION
**Pull request by Sophyia7**
I think this project is a good fit for Up For Grabs. Let me know if you need anything else.

### CODE REVIEW

File: _data/projects/pieces-example-ts.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Pieces Example TS
+desc: This is an example project of getting started with Pieces OS ts package.
+site: https://github.com/pieces-app/example-ts
+tags:
+  - good first issue
+  - bug
+  - documentation
+  - community review
+  - design
```
Comment (shiftkey) [inline _data/projects/pieces-example-ts.yml:9]: We don't allow spaces in tags, which is why there's parsing issues with this YAML file. 

Unless a tag is relevant to the project, I'd suggest removing these for now.

**Review by gdamiaN538 — COMMENTED** [review]
#4477

**Review by gdamiaN538 — APPROVED** [review]
> __

**Comment by shiftkey** [comment]
Closing this out due to inactivity. Please submit a new PR if you still wish to see your project listed on the site...

**Pull request closed without merging**